tests/lib/tests.py

```diff
@@ -122,6 +123,23 @@ def run_tests(path, conn, replaces, dbtype, debugging=False, unattended_debuggin
     """
     files = sorted(glob(path))
     tests = {'total': 0, 'ok': 0, 'errors': 0}
+    
+    # Set client_min_messages for PostgreSQL connections
+    if dbtype == 'postgresql':
+        valid_levels = ['DEBUG5', 'DEBUG4', 'DEBUG3', 'DEBUG2', 'DEBUG1', 'LOG', 'NOTICE', 'WARNING', 'ERROR']
+        client_min_messages = os.environ.get('POSTGRES_CLIENT_MIN_MESSAGES', 'DEBUG3')
```

Comment:
Is there a reason for getting the value via an environment variable and not with a parameter at https://github.com/tds-fdw/tds_fdw/blob/01eec666fcace11a3bcfc30edb07693934ce624d/tests/postgresql-tests.py? Maybe a new `--postgres_debug` parameter?

I'd even keep debugging disabled by default. Usually you want to enable it specifically if your tests are failing and the default logs are not enough.

Then I'd enable it by default for tests on PRs, but I'd keep it disabled for the regular tests on the `master` branch (with a checkbox to enable debugging specifically).

Reply:
> > Then I'd enable it by default for tests on PRs, but I'd keep it disabled for the regular tests on the master branch (with a checkbox to enable debugging specifically).
> 
> Before we change it for PRs in Jenkins, I guess we should have the command-line parameter in master, right?

This has been handled by #407 / #408 .